velox/common/file/FileSystems.h

```diff
@@ -57,6 +57,10 @@ class FileSystem {
   /// Returns the name of the File System
   virtual std::string name() const = 0;
 
+  virtual std::string_view extractPath(std::string_view path) {
```

Comment:
Because `select "$" path from table` returns such format
file:/Users/ke1024/Desktop/hive_data/tpch/orders/20230315_002531_00001_4u2ij_ee0a6f47-843a-4965-b93a-f06a396dfcb4

Reply:
Let's add comment
```
/// Returns the file path without the fs scheme prefix such as "local:" prefix for local file system.
```